### ISSUE TITLE
requires 3d?

### ISSUE THREAD

**Issue by augustogoncalves** [libvarun/forge-extensions#5]
for usability, can we add a config option to only load in 2D, or 3D, or both? for instance, the current Icon Markup works only in 3d... 

requires: 2d|3d

not sure if that's a feature or property on the built-in extensions, explode is an example, can you check?

**Comment by libvarun** [comment]
when developer emits event to load an extension, they pass the viewer instance along with extension id. so they have control over which in which viewer to load the extension.
https://github.com/libvarun/StandardExtensions/blob/master/public/js/ForgeViewer.js#L41

Need to add whether the extension works in 3d or 2d or both

**Comment by augustogoncalves** [comment]
yes, right, just want to disable extensions on the list if not applicable
